org.osate.ui/src/org/osate/ui/wizards/NewModelWizard.java

```diff
@@ -95,7 +95,7 @@
  * @author jseibel
  */
 abstract class NewModelWizard extends Wizard implements INewWizard {
-	private static final String[] HIDE_FOLDERS = { "instances", "diagrams", "imv" };
+	private static final String[] HIDE_FOLDERS = { ".aadlbin-gen", ".settings", "instances", "diagrams", "imv" };
```

Comment:
Should hide all directories that start with '.'

Reply:
Done.